additional-functions/vat-codes.md

```diff
@@ -0,0 +1,16 @@
+## VAT codes
```

Comment:
I'd eliminate this section and rather improved the parts inside documentation corresponding to Tax environment, Taxation, tax rate and amount.